src/test/java/bdd/runner/CucumberTestRunner.java

```diff
@@ -0,0 +1,17 @@
+package bdd.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(
+        plugin = {"pretty", "html:target/cucumber-reports",
+                "json:target/cucumber-reports/CucumberTests.json",
+                "testng:target/cucumber-reports/CucumberTests.xml"},
+                //"io.qameta.allure.cucumberjvm.AllureCucumberJvm"},
```

Comment:
commented code

Reply:
Deleted, thank you for pointing.
Please see in the following commit.